src/components/ToDoInfo/ToDoInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { User } from '../../types/User';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  user: User | null,
+  title: string,
+  completed: boolean,
+};
+
+export const ToDoInfo: React.FC<Props> = ({
+  user,
+  title,
+  completed,
+}) => (
+  <>
+    <h2 className="item__title">
+      {'Task: '}
+      <i>{title}</i>
+    </h2>
+
+    <label>
+      Done:
+      <input type="checkbox" checked={completed} className="item__checkbox" />
```

Comment:
```suggestion
      <input type="checkbox" defaultChecked={completed} className="item__checkbox" />
```

`checked` is permanent, so i can't change the checkboxes states now
`defaultChecked` just means the start `checked` value with possibitly to change it

Reply:
👍